airbyte-webapp/src/components/StatusIcon/StatusIcon.tsx

```diff
@@ -20,20 +20,22 @@ interface StatusIconProps {
 
 const getBadgeWidth = (props: StatusIconProps) => (props.big ? (props.value ? 57 : 40) : props.value ? 37 : 20);
 
-const _iconByStatus: Partial<Record<StatusIconStatus, IconDefinition | undefined>> = {
+const _iconByStatus = {
```

Comment:
:information_source: By changing this just to a `as const`, TypeScript understands which keys exactly exist in this, unless beforehand where it always suspected accessing something in here could also be `undefined`.